multimodal/src/autogluon/multimodal/constants.py

```diff
@@ -9,6 +9,7 @@
 IMAGE_PATH = "image_path"
 IMAGE_BYTEARRAY = "image_bytearray"
 IDENTIFIER = "identifier"
+DOCUMENT = "document"
```

Comment:
Is document a good name for this type?  Document sounds like pure text. How about `doc_image`?

Reply:
not all document are image, we will support pdf and  perhaps other types of document later.